python/tank/util/path.py

```diff
@@ -59,3 +63,41 @@ def add_path_to_env_var(env_var_name, path, prepend=False):
             paths.append(path)
     # and put it back
     os.environ[env_var_name] = env_var_sep.join(paths)
+
+
+def ensure_path_exists(path):
```

Comment:
Maybe this should be part of the internal API for the time being? (see discussion in the hook code)